tests/functional/docs/test_help_output.py

```diff
@@ -21,15 +21,15 @@
 (it's easier to verify).
 
 """
-import io
 import os
 
 from awscli.testutils import BaseAWSHelpOutputTest
 from awscli.testutils import FileCreator
 from awscli.testutils import mock
 from tests import CLIRunner
+from awscli.testutils import aws
```

Comment:
What's this import for?

Reply:
It looks like this got added automatically when cherry-picking the v1 commit onto v2. It looks like Kyle rewrote the [original v1 test](https://github.com/aws/aws-cli/commit/9b3b86c58eadb96023698f84f86fefc01b70c171#diff-07670404753d70f985622d5db67d414d39dd65720a7bbdfb41b0e56e92a9971dR492) in v2 to use `CLIRunner` instead of `aws`. I'll remove it.